### ISSUE TITLE
BART: Denoising Sequence-to-Sequence Pre-training for Natural Language Generation, Translation, and Comprehension

### ISSUE THREAD

**Issue by dltmddbs100** [dltmddbs100/Paper-log#19]
## Main Idea
**A universal language model specialized in various fields**
Paper: <https://arxiv.org/abs/1910.13461>

## Abstract
We present BART, a denoising autoencoder for pretraining sequence-to-sequence models. BART is trained by (1) corrupting text with an arbitrary noising function, and (2) learning a model to reconstruct the original text. It uses a standard Tranformer-based neural machine translation architecture which, despite its simplicity, can be seen as generalizing BERT (due to the bidirectional encoder), GPT (with the left-to-right decoder), and many other more recent pretraining schemes. We evaluate a number of noising approaches, finding the best performance by both randomly shuffling the order of the original sentences and using a novel in-filling scheme, where spans of text are replaced with a single mask token. BART is particularly effective when fine tuned for text generation but also works well for comprehension tasks. It matches the performance of RoBERTa with comparable training resources on GLUE and SQuAD, achieves new stateof-the-art results on a range of abstractive dialogue, question answering, and summarization tasks, with gains of up to 6 ROUGE. BART also provides a 1.1 BLEU increase over
a back-translation system for machine translation, with only target language pretraining. We also report ablation experiments that replicate other pretraining schemes within the BART framework, to better measure which factors most influence end-task performance.

## Contribution
+ BERT의 bi-directional한 encoder와 GPT의 auto-regressive한 특성을 모두 가진 encode-decoder 모델 BART 제시
+ BART는 encoder의 input과 decoder의 input이 동일하지 않아도됨 => 다양한 noise function의 사용이 가능
+ Pre-training 시 적용가능한 다양한 noise function을 실험
 [Token Masking, Token Deletion, Text Infilling, Sentence Permutaion, Document Rotation]
 => Token masking이 가장 중요 / Generation task의 경우 deletion이 효과적
+ Discriminative task에서 RoBERTa에 준하는 성능
+ Generation task에서의 높은 성능은 discriminative task와 trade-off를 가지지 않음
+ 여러 task에 보편화될 수 있는 범용성이 강함
+ 특히 요약시 pretrain된 내용에 대한 사전정보들을 이용해 input에 대한 정보와 결합하여 요약을 생성
+ 자연어에 대한 이해와 생성간의 관계를 올바르게 학습함

## Reviews
<https://deepkerry.tistory.com/32>
